Library/Homebrew/cmd/uninstall.rb

```diff
@@ -73,5 +78,7 @@ def uninstall
         force:   args.force?,
       )
     end
+
+    Cleanup.autoremove if Homebrew::EnvConfig.autoremove?
```

Comment:
```suggestion
    Cleanup.autoremove
```

any move the guard into the method, maybe, given we're guarding every call?

Reply:
Nope, the call to `Cleanup#autoremove` in the autoremove subcommand doesn't use the guard clause.

Reply:
Ah, yes, ignore me!